chapters/ch32.md

```diff
@@ -1,49 +1,47 @@
 
 
 > Suppose you have a very small training set of 100 examples. You train your algorithm using a randomly chosen subset of 10 examples, then 20 examples, then 30, up to 100, increasing the number of examples by intervals of ten. You then use these 10 data points to plot your learning curve. You might find that the curve looks slightly noisy (meaning that the values are higher/lower than expected) at the smaller training set sizes.
 
---> _replace THIS LINE by your translation for the above line_
-
+Giả sử bạn có một tập huấn luyện rất nhỏ gồm 100 mẫu. Bạn huấn luyện thuật toán của mình bằng cách sử dụng một tập hợp con được chọn ngẫu nhiên gồm 10 mẫu, sau đó 20 mẫu, sau đó 30, lên đến 100, tăng số lượng mẫu theo các khoảng mười. Sau đó, bạn sử dụng 10 điểm dữ liệu này để vẽ đồ thị quá trình học. Bạn có thể thấy rằng đồ thị  trông hơi nhiễu(có nghĩa là các giá trị cao hơn/thấp hơn dự kiến) ở kích thước tập huấn luyện nhỏ hơn.
 
 > When training on just 10 randomly chosen examples, you might be unlucky and have a particularly “bad” training set, such as one with many ambiguous/mislabeled examples. Or, you might get lucky and get a particularly “good” training set. Having a small training set means that the dev and training errors may randomly fluctuate.
 
---> _replace THIS LINE by your translation for the above line_
-
+Khi huấn luyện chỉ với 10 mẫu được chọn ngẫu nhiên, bạn có thể không may mắn và có một tập huấn luyện đặc biệt “xấu”, chẳng hạn như một tập có nhiều mẫu không xác định/bị gán nhãn sai. Hoặc, bạn có thể gặp may mắn và nhận được một tập huấn luyện đặc biệt “tốt”. Có một tập huấn luyện nhỏ đồng nghĩa với lỗi trên tập phát triển và tập huấn luyện có thể dao động ngẫu nhiên.
 
 > If your machine learning application is heavily skewed toward one class (such as a cat classification task where the fraction of negative examples is much larger than positive examples), or if it has a huge number of classes (such as recognizing 100 different animal species), then the chance of selecting an especially “unrepresentative” or bad training set is also larger. For example, if 80% of your examples are negative examples (y=0), and only 20% are positive examples (y=1), then there is a chance that a training set of 10 examples contains only negative examples, thus making it very difficult for the algorithm to learn something meaningful.
 
---> _replace THIS LINE by your translation for the above line_
-
+Nếu ứng dụng học máy của bạn bị lệch nhiều về một lớp (chẳng hạn như nhiệm vụ phân loại mèo ở đó số mẫu âm --không phải mèo-- lớn hơn nhiều số mẫu dương --là mèo) hoặc nếu nó có một số lượng lớn các lớp (chẳng hạn như nhận dạng 100 loài động vật khác nhau) , khi đó cơ hội lựa chọn một lớp “không đại diện” đặc biệt hoặc tập huấn luyện xấu cũng lớn hơn. Ví dụ: nếu 80% mẫu của bạn là mẫu âm (y = 0) và chỉ 20% là mẫu dương (y = 1), thì có khả năng một tập huấn luyện gồm 10 mẫu chỉ chứa các mẫu âm, và rất khó để thuật toán học được điều gì đó có ý nghĩa
```

Comment:
Đoạn này bản gốc cũng viết hơi dài và khi bạn thêm giải thích vào trong bản dịch thì nó lại càng khó đọc hơn. Bạn có thể tách ra làm 2 câu và chỉnh lại để ý nghĩa của bản dịch chau chuốt hơn:

- Nếu ứng dụng học máy của bạn bị lệch nhiều về một lớp: chẳng hạn như bài toán phân loại mèo có số mẫu âm (không phải mèo) lớn hơn nhiều số mẫu dương (là mèo). Hoặc, nếu bài toán có một số lượng lớn các lớp (chẳng hạn như nhận dạng 100 loài động vật khác nhau) , khi đó cơ hội lựa chọn một lớp “không đại diện” đặc biệt hoặc tập huấn luyện xấu cũng lớn hơn.

Reply:
ok a.